### ISSUE TITLE
Include last modify date in rule

### ISSUE THREAD

**Issue by brad2014** [jzdziarski/littleflocker#56]
Enhancement idea: include (optional?) last modified date in rule syntax, and present it in the rules window.

```
allow path  "/mypath" "command" rwc 20160905T120000Z
```

Example uses: By sorting by LMD, I can see the group of rules that I created during a particular session or activity, and undo or adjust them as a group.  When two rules are similar, I can tell which one was added later (useful in choosing how to merge overlapping rules).

**Comment by jzdziarski** [comment]
I like the idea of grouping rules; i'm not sure import time is the way to go; especially something like Xcode, it takes several uses of it before you get all the rules. Perhaps a way to tag them might make more sense. That way you can work with groups that you didn't originally import.

**Comment by brad2014** [comment]
Indeed, over time, I can see you might want to dress rules in various ways. One might end up extending the rule line parsing to support named parameters, e.g. from:
```
allow prefix "$HOME/" "/bin/bash via any" rmc
```
to also accept something (à la getopt) that could simply but extensibly be parsed into a dict:
```
allow --app="/bin/bash" --via="any" --mode="rwc" --prefix="$HOME/" 
  --tag="xcode"  --creator="brad2014" --mtime="20150908T190000Z" 
  --disable --tag="development" ...
```
This would also get rid of the "reserved word" problem from #57.  Future named parameters could be easily added without breaking older rule files. 

Con: Of course, it would make the rule file bigger (and it'll never have as many flags as "ls", so why even try?)
